src/methods/extract.jl

```diff
@@ -5,7 +5,7 @@ istrue(::_True) = true
 istrue(::_False) = false
 
 """
-    extract(x, data; atol)
+    extract(x, data; [geometry, index, name, skipmissing, atol])
```

Comment:
```suggestion
    extract(x, data; kw...)
```

At this point people can look at the Keywords section